src/kansei/fallen/node/fallen_determinant.cpp

```diff
@@ -30,26 +30,45 @@
 namespace kansei::fallen
 {
 
-FallenDeterminant::FallenDeterminant(const DeterminantType & type)
-: fallen_status(FallenStatus::STANDUP), determinant_type(type), fallen_back_raw_limit(491.0),
-  fallen_front_raw_limit(458.0), fallen_right_raw_limit(519.0), fallen_left_raw_limit(498.0)
+FallenDeterminant::FallenDeterminant(const std::string &fallen_type)
+: fallen_status(FallenStatus::STANDUP)
 {
+  if (fallen_type == "orientation") {
+    determinant_type = DeterminantType::ORIENTATION;
+  } else if (fallen_type == "accelero") {
+    determinant_type = DeterminantType::ACCELERO;
+  } else {
+    throw std::runtime_error("Invalid fallen type: " + fallen_type);
+  }
 }
```

Comment:
I suggest keeping the usage of `DeterminantType` here instead of `std::string`.

It appears that `std::string` is being used to allow input from the console to construct this class. Instead of modifying the parameter of the constructor to use `std::string`, let's parse the input from the console to be `DeterminantType` instead.